index.html

```diff
@@ -227,5 +235,29 @@ <h1>Skills</h1>
             </div>
         </div>
     </section>
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form action="https://formspree.io/f/xrgvqloo" method="POST">
+                <input type="text" name="tect" maxlength="30" placeholder="Full name" required>
+                <input type="email" name="email" placeholder="Email adress" required>
+                <textarea maxlength="500" name="message" placeholder="Enter text here" required></textarea>
+                <button type="submit" name="submit">Get in touch</button>
```

Comment:
- [ ] Good job trying to implement the hover state of the buttons in your application. KIndly note that your hover state should be this 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/220443480-607fd881-4851-42ca-94ab-ada407882b6b.png)

but instead, you were using this 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/220443566-7e2653af-3926-4080-8376-f245e206b39a.png)

Kindly fix it 💯